<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>224</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="ledC">
       <property name="text">
        <string>V 3.3/1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="fanC">
       <property name="text">
        <string>Ki 77/10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bulbC">
       <property name="text">
        <string>Kd 88/0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>PID</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>231</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="RS">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="GS">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="BS">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>233</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comL"/>
      </item>
      <item>
       <widget class="QPushButton" name="openB">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeB">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Kp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ki</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Kd</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Set voltage</string>
    </property>
    <widget class="QDial" name="servoK">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>50</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0-   V    -5</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="textF">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendB">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>151</y>
      <width>38</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="tempL">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="tempB">
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="lcdN">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="circle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>431</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Set voltag–µ 0-5 V</string>
    </property>
   </widget>
   <zorder>groupBox_4</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>textF</zorder>
   <zorder>sendB</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>lcdN</zorder>
   <zorder>circle</zorder>
   <zorder>graph</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
